webpack.config.js

```diff
@@ -30,3 +24,33 @@ module.exports = {
     },
   },
 };
+
+const nodeConfig = {
+  ...baseConfig,
+  target: "node",
+  output: {
+    library: {
+      type: "umd",
```

Comment:
```suggestion
      type: "commonjs",
```
I'm not sure there's any reason to use umd. If a consumer of the library wants to use the script directly in a browser, they can import the ES module. If a consumer wants to use `require` in node, they can require the commonjs module.

Reply:
They'll get the .mjs file.